### PR TITLE
feat: highlighted views and queries

### PR DESCRIPTION
**Pull request by capJavert**
Implementation [from](https://dailydotdev.slack.com/archives/C06S6HEUZB9/p1714918400969439)

- [ ] see which columns need indexes

### CODE REVIEW

File: __tests__/cron/updateHighlightedViews.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { crons } from '../../src/cron/index';
+import cron from '../../src/cron/updateHighlightedViews';
+
+describe('updateHighlightedViews cron', () => {
+  it('should be registered', () => {
```
Comment (sshanzel) [inline __tests__/cron/updateHighlightedViews.ts:5]: I like this. Registering the worker gets missed sometimes and this would ensure they got listed.

**Review by sshanzel — APPROVED** [review]
I think it would be a good idea to always include a test for registering the worker, moving forward. Looks good to me 🚀

**Review by nensidosari — APPROVED** [review]

**Review by idoshamun — APPROVED** [review]
I haven't fully compared the queries here to my suggestion so just double check they match

File: __tests__/cron/updateHighlightedViews.ts (hunk 1 above)
Comment (capJavert) [reply __tests__/cron/updateHighlightedViews.ts:5]: Yep, all the recent crons/worker have this 👌

**Comment by rebelchris** [comment]
@idoshamun // @omBratteng // @nensidosari I've now added all queries/test and cleaned up unused field in MVs
Could at least one of you sanity check before I merge? 🙏

**Review by omBratteng — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]
Looks great

**Pull request merged**